packages/cli/src/commands/network/parameters.test.ts

```diff
@@ -0,0 +1,117 @@
+import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import { testLocally } from '../../test-utils/cliUtils'
+import Parameters from './parameters'
+process.env.NO_SYNCCHECK = 'true'
+
+testWithGanache('network:parameters', () => {
+  test('runs', async () => {
+    const spy = jest.spyOn(console, 'log')
+    await testLocally(Parameters, [])
+    expect(spy.mock.calls).toMatchInlineSnapshot(`
```

Comment:
what about using https://github.com/celo-org/developer-tooling/blob/d0f13daaa2d879debc74a6ee27742b95550fccec/packages/cli/src/test-utils/cliUtils.ts#L19?

the snapshot would become much more readable then